google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/functions/WindowsPasswordGenerator.java

```diff
@@ -131,34 +133,50 @@ public String apply(AtomicReference<Instance> instance) {
                   operation.get().httpErrorMessage());
          }
 
-         // Retrieve the result from the last line of serialPortOutput; expect JSON string
+          // TODO remove me
+          try {
+              Thread.sleep(3 * 60 * 1000);
```

Comment:
This is an important line.
If it is 30 seconds the response looks like:
```json
contents": "{\"ready\":true,\"version\":\"Microsoft Windows NT 6.2.9200.0\"}"
```

If it is longer we have what we need :)
```json
contents": "{\"ready\":true,\"version\":\"Microsoft Windows NT 6.2.9200.0\"}\n{\"encryptedPassword\":\"DclUPv7Lbf5W7gRw7zg85+jlxzHo+SASgMgA1OkZmQEsv2DUZHqtldnEQmmBQrmFag91YYg8pO2O87Vki406tApqfGxW81AXv99N\\/StgDjrUHoQund6Sqj2etJz+x3prYqYZdRKdeGuT9rNPLVeta77IDLxuVUG5C9Q378QmWJ4qNySrtck0pZwqW90n+t7zdTA5h3mzy4brtStkZtcsQBCI6b7wiAimqejbmfHP5HnebVz+GHmp4WE5xvN9EESrzVzVjGGpGBVixt3D15HaMeJfz15YAIa9xHuUPAzBcgUSgIb66wQDpxu3UIzKcBK4\\/3SCIr9C5La0n5OJ4Zqcew==\",\"exponent\":\"AQAB\",\"modulus\":\"h3uacuDZ8zgUx9tjE57L7oSOB3xYvOsML9hqOJbv9CPVHS6GiY4zB5D8oGFZf5O4iZwVCZnGFCaT8ft7DEL1dvNekBWbnKRN+MdMs4aCrvAoNZX\\/p4BNFE9ZkagMiToc9CDyNjn94LRQE0zkNDN\\/le7Nkk1aMk7KMXa5AtGvwQyEe5hUYrjk+Dua6RwCglAX1pROPupdT6t3H7ChgkGPPB8TJ5pp38N6ieXfK0LeptiqhgLfTfYeeNtUCgIBM50l9Hq0ejeLmC87rqWK9Ct8ePckvK7Al1rozu10SW3+jwCXZl2G9nkyuW+M9cd39IzkK8Vuf4aoXgUuDhBAVig\\/jQ==\",\"passwordFound\":true,\"userName\":\"Administrator\"}"
```